packages/opentelemetry-types/src/trace/tracer.ts

```diff
@@ -76,7 +78,7 @@ export interface Tracer {
    *
    * @todo: Change return type once BinaryFormat is available
```

Comment:
Could you please remove this `@todo` tag?

Reply:
Oh yes sorry for that